test/main.cpp

```diff
@@ -0,0 +1,13 @@
+#include <catch2/catch_test_macros.hpp>
+#include <catch2/matchers/catch_matchers_floating_point.hpp>
+
+TEST_CASE("Catch2 should work") {
```

Comment:
LOL and who said they didn't _love_ macro hacks???? 😂

Reply:
Well, it's a necessary evil for unit testing frameworks that do all sorts of black magic behind the scenes.